### ISSUE TITLE
sanipack command error on Windows 10

### ISSUE THREAD

**Issue by mamfi-0221** [rexxars/sanipack#6]
Hi @rexxars,

I have a plugin created from **sanipack** on my local terminal where running either "**_yarn watch_**" , "**_npm run watch_**" or "**_npm run build_**" commands all return an "`[error] spawn babel ENOENT`" message. 

These are the scripts from the package.json file of the said plugin:
"scripts": {
    "build": "sanipack build",
    "watch": "sanipack build --watch",
    "prepublishOnly": "sanipack build && sanipack verify"
 },

I tried running the commands manually but to no luck. 

Environment: Windows 10 Home (OS Build 19041.928), VS Code v.1.56.0 (used its powershell terminal)

Steps to reproduce:
1. Have created sanipack plugin on local terminal
2. Go to the folder of the sanipack plugin
3. Run `yarn watch` , `npm run watch` or `npm run build`

Actual result: running the command returns the following error messages
`[info] � Watching for changes
[error] spawn babel ENOENT
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

Expected result : Plugin successfully compiles

I would really appreciate your help on this. Thanks a lot!

**Comment by dorelljames** [comment]
My other colleagues are reporting this as well. This seems to happen on Windows OS only. Not sure why...

**Comment by rexxars** [comment]
I don't have access to a windows terminal right now, but would love contributions

**Comment by rexxars** [comment]
Could you check if this is fixed in v2.1.0?

**Comment by mamfi-0221** [comment]
Hi @rexxars,

My apologies, I just got to check this again today. 
I updated my `sanipack` version to 2.1.0 and although running the `yarn build` or `yarn watch` commands still throws the same error on my Windows 10 terminal, the `sanipack commands` are already working.

a. `sanipack build`
![image](https://user-images.githubusercontent.com/78787873/138028218-912a417a-9802-413d-9d62-929d26dd6ed8.png)

b. `sanipack build --watch` (also works: `sanipack build --watch --verbose`)
![image](https://user-images.githubusercontent.com/78787873/138028281-3802305a-9d11-47e3-871f-31e028bb9adb.png)

Not sure why the yarn commands failed on Windows but nonetheless, I'll get to use the sanipack commands as needed now without the problems I initially reported. 

Thank you so much for your help! :)
